### PR TITLE
Ocs mrogers about assets subbranch

### PR DESCRIPTION
**Pull request by osimarkrogers**
About assets content for review

### CODE REVIEW

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # About assets
 
-Content to be developed
\ No newline at end of file
+---
+uid: Aboutassets
+---
+
+Assets are objects used to represent physical entities in OCS.  Assets typically represent devices with multiple data streams, although you can create an asset with only one stream if needed.
```
Comment (lasato-OSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:7]:
```suggestion
Assets represent physical entities in OCS.  Typically, they are configured with multiple data streams, although you can also create an asset with a single stream.
```

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+An asset is a container for data streams associated with a particular device or entity. When you create an asset, you scope it to a namespace and a set of streams within that namespace. You can create assets on an ad hoc basis or create them from an asset type, a template for creating similar assets.
```
Comment (lasato-OSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:9]:
```suggestion
An asset is a container for data streams associated with a particular device or entity. When you create an asset, you scope it to a namespace and a set of streams within that namespace. Assets typically represent devices with multiple streams. However, you can also create assets with a single stream. You can create assets on an ad hoc basis or create them from an asset type, a template for creating similar assets.
```

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+When you create assets in OCS, you can use the Asset Explorer in the OCS Portal to visualize selected data streams and properties for troubleshooting and analysis of the associated devices or entities.
```
Comment (lasato-OSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:11]:
```suggestion
You can use the asset explorer in the OCS Portal to create assets and visualize selected data streams and properties for troubleshooting and analysis of the associated devices or entities.
```

Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md]: Done.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md (hunk 2 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md]: Done.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md (hunk 1 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md]: Done.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+---
+uid: Aboutassets
+---
+
+An asset is a container for data streams associated with a particular device or entity. When you create an asset, you scope it to a namespace and a set of streams within that namespace. Assets typically represent devices with multiple data streams. However, you can create an asset with only one stream if needed. 
+You can create assets on an ad hoc basis or create them from an asset type, a template for creating similar assets.
+
+Use the asset explorer in the OCS Portal to create assets and visualize selected data streams and properties for troubleshooting and analysis of the associated devices or entities.
+
+For more information on assets, see [Assets](xref:WhatOCSdoes#assets)
```
Comment (afloresOSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:12]: Need a period at the end of the sentence.
```suggestion
For more information, see [Assets](xref:WhatOCSdoes#assets).
```

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
 # About assets
 
-Content to be developed
\ No newline at end of file
+---
+uid: Aboutassets
+---
+
+An asset is a container for data streams associated with a particular device or entity. When you create an asset, you scope it to a namespace and a set of streams within that namespace. Assets typically represent devices with multiple data streams. However, you can create an asset with only one stream if needed. 
+You can create assets on an ad hoc basis or create them from an asset type, a template for creating similar assets.
```
Comment (afloresOSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:8]: Combine with the sentence in the next paragraph.
```suggestion
Use the asset explorer in the OCS Portal to create assets and visualize selected data streams and properties for troubleshooting and analysis of the associated devices. You can create assets on an ad hoc basis or create them from an asset type, which acts as a template for creating similar assets.
```

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 6]
```diff
@@ -1,3 +1,12 @@
 # About assets
 
-Content to be developed
\ No newline at end of file
+---
+uid: Aboutassets
+---
+
+An asset is a container for data streams associated with a particular device or entity. When you create an asset, you scope it to a namespace and a set of streams within that namespace. Assets typically represent devices with multiple data streams. However, you can create an asset with only one stream if needed.
```
Comment (afloresOSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:7]:
```suggestion
An asset is a container for data streams associated with a particular device or entity. When you create an asset, you scope it to a namespace and a set of streams within that namespace. Assets typically represent devices with multiple data streams; however, you can create an asset with only one stream if needed. 
```
what is an entity? or at least an example of one? "... or entity, such as a ???."

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 7]
```diff
@@ -1,3 +1,12 @@
 # About assets
```
Comment (afloresOSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:1]: We had a standard for the CMS that we would never begin a topic with "About". I realize that the current OCS help has a bunch of topics that start in this manor, but it is going to make it difficult to search and is unnecessary.

**Review by afloresOSI — CHANGES_REQUESTED** [review]

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md (hunk 7 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:1]: @lasato-OSI Perhaps we should discuss this at our next OCS Writers meeting.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md (hunk 6 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:7]: I think "object" works better.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md (hunk 4 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:12]: Good catch.  Done.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md (hunk 5 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:8]: Done.

File: Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md [hunk 8]
```diff
@@ -1,3 +1,11 @@
...
 
-Content to be developed
\ No newline at end of file
+---
+uid: Aboutassets
+---
+
+An asset is a container for data streams associated with a particular device or object. When you create an asset, you scope it to a namespace and a set of streams within that namespace. Assets typically represent devices with multiple data streams; however, you can create an asset with only one stream if needed.
```
Comment (lasato-OSI) [inline Content_Portal/add-organize-data/attach-meaning/assets/about-assets.md:7]:
```suggestion
An asset is a container for multiple data streams. When you create an asset, you scope it to a namespace and a set of streams within that namespace. 
```
Mark, why don't we remove the mention of device/object. I looked back at Victor's feedback on what you wrote about assets for the overview topic, and he said, "No need to mention "a single device" or "multiple devices". The end users who create these digital twin may or may not possess knowledge of the sensors/IoT devices. The key is that an asset can consists of multiple streams."

**Review by lasato-OSI — APPROVED** [review]
We may need to revisit this later, but it's fine for now. Thanks.

**Review by afloresOSI — APPROVED** [review]

**Pull request merged**